numba/unicode_support.py

```diff
@@ -311,6 +311,12 @@ def _PyUnicode_IsCaseIgnorable(ch):
     return ctype.flags & _PyUnicode_TyperecordMasks.CASE_IGNORABLE_MASK != 0
 
 
+@register_jitable
+def _PyUnicode_IsSpace(ch):
```

Comment:
Reference with SHA?

Reply:
There is no such functionality in https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Objects/unicodectype.c. So I'm not able to insert a reference here.

Reply:
Try this:
https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Include/unicodeobject.h#L125-L126
also, you'll note it has an ascii fast-path you may wish to implement.